metabase/metabase
# Upgrade chromatic and sync storybook packages

### Description

Upgrade chromatic and sync storybook packages to avoid duplications and to bypass CI issues like this https://github.com/metabase/metabase/actions/runs/7336443587/job/19975769824?pr=37013

Chromatic became broken after https://github.com/metabase/metabase/pull/37007 because of using different versions of storybook packages (both 6.5.15 and 6.5.16 were in packages.json)

### How to verify

- make sure chromatic deployment works
- `yarn build-storybook` should work
- `yarn storybook` should work

### Checklist

~- [ ] Tests have been added/updated to cover changes in this PR~
